### DISCUSSION TITLE
No Sound through speakers on HP Laptop

### DISCUSSION THREAD

**Discussion by OldMichaelC** [thesofproject/sof#7497] [Q&A]
1. I am a linux newbie....be gentle with me please.
2. Because of 1...please answer like I am an idiot...I will not be offended.
3. Actual Question/Issue:   Laptop is an HP Envy 17" Laptop model cr0013dx. I have the same issue as many others I have seen in different forums but I have not found an answer to my issue that works. Basically no sound on the laptop speakers. Everything "looks" like it is working right, soundcard shows up in settings, when I test sound it looks like its doing everything, except no sound through speakers. Bluetooth headphones and speakers work perfectly...but not the on board speakers. As I understand it, this is something worked on with this group, so thought I would ask for help here.

**Comment by plbossart** [comment]
@OldMichaelC  this "Discussions" tab is intended for developers, not support issues.

please read https://thesofproject.github.io/latest/getting_started/index.html#debug-audio-issues-on-intel-platforms

and file an issue here https://github.com/thesofproject/linux/issues if you have no found the solution to your problem

please attach the log of 'alsa-info' so that can figure out what your device is.
